test/jsApi.yml

```diff
@@ -0,0 +1,130 @@
+- test: Test js input
+  steps:
+    -   in: bn create node8 returnInput
+    -   in: |-
+            echo 'exports.handler = (input) => ({ inputType: typeof input, input});' >function.js
+    -   in: bn deploy returnInput
+    -   in: curl -s "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{}}'
+    -   in: curl -s --data '{"' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"undefined"}'
+    -   in: curl -s --data '{}' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{}}'
+    -   in: curl -s --data '{"foo":"bar","yet":123}' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
```

Comment:
nit: might be cleaner to alias `https://${BINARIS_INVOKE_ENDPOINT}/v1/run/` as `BINARIS_RUN_ENDPOINT`.